src/config/keys.ts

```diff
@@ -3,6 +3,7 @@ import type { UUID } from '../types';
 export const ITEMS_KEY = 'items';
 export const OWN_ITEMS_KEY = [ITEMS_KEY, 'own'];
 export const buildItemKey = (id: UUID) => [ITEMS_KEY, id];
+export const MULTIPLE_ITEMS_KEY = [ITEMS_KEY, 'multiple']
```

Comment:
The reason we have this key is to avoid refetching the items twice. In this case, fetching id1+id2 and fetching id1+id3  would collide. For instance, you fetch id1+id2, it will be stored in `[items, multiple]`. Then you will want to fetch id1+id3, but `[items, multiple]` would be still valid and thus  React-query won't fetch your items. To avoid this behavior, you need the key to depend on the items you are fetching.